src/backend/app/projects/project_schemas.py

```diff
@@ -50,26 +50,33 @@ class BETAProjectUpload(BaseModel):
     xform_title: Union[str, None]
     odk_central: ODKCentral
     hashtags: Union[List[str], None]
-    organisation_id: int = None
+    organisation_id: Optional[int]
     # city: str
     # country: str
 
 
+class Feature(BaseModel):
+    id: int
+    project_id: int
+    task_id: Optional[int]
+    geometry: Optional[Feature]
+
+
 class ProjectSummary(BaseModel):
     id: int = -1
     priority: ProjectPriority = ProjectPriority.MEDIUM
     priority_str: str = priority.name
-    title: str = None
-    location_str: str = None
-    description: str = None
-    num_contributors: int = None
-    total_tasks: int = None
-    tasks_mapped: int = None
-    tasks_validated: int = None
-    tasks_bad: int = None
-    hashtags: List[str] = None
-    organisation_id: int = None
-    organisation_logo: str = None
+    title: Optional[str]
```

Comment:
I haven't gone through all of the docs for Pydantic V2, but I think the syntax might be:

`title: Optional[str] = None` as Optional params still need a default value.